### ISSUE TITLE
Probably missing maxOccurs in PlannedPatternDelivery > PlannedPattern > Via

### ISSUE THREAD

**Issue by realholgi** [ITxPT/S02#3]
The max occurs of Via is not specified which defaults to 1. According to the specification it should be unbound.

**Specificiation:**
https://wiki.itxpt.org/index.php?title=S02P06-AVMS_-_v2.1.0

![grafik](https://user-images.githubusercontent.com/980419/108494012-54a2ea80-72a7-11eb-8cb4-bf10baf0f9fe.png)

Current XSD Schema https://github.com/ITxPT/S02/blob/master/xsd/S02P06_avms/PlannedPatternDelivery.xsd

```xml
<xs:element minOccurs="0" ref="OriginTtsName"/>
<xs:element ref="Via" minOccurs="0"/>
<xs:element minOccurs="0" ref="DestinationPlaceRef"/>
```

Should be

```xml
<xs:element minOccurs="0" ref="OriginTtsName"/>
<xs:element ref="Via" minOccurs="0" maxOccurs="unbounded"/>
<xs:element minOccurs="0" ref="DestinationPlaceRef"/>
```

**Comment by JoaoDiogoF** [comment]
@victorDARDENNE I see you added a bug. Can you tell us where it is actually correct, in the specification or in the xsd schema?

**Comment by realholgi** [comment]
I the XSD is wrong. The spec is correct, as i think there should be many *possible *Via** elements for a pattern (e.g the bus can to Hamburg via Frankfurt, Leipzig AND Berlin).

**Comment by victorDARDENNE** [comment]
> @victorDARDENNE I see you added a bug. Can you tell us where it is actually correct, in the specification or in the xsd schema?

This issue points a gap between the spec and the XSD. When the tag 'Bug' is used, it means that it is a bug related to the repo content, in that case the PlannedPatternDelivery XSD.
Otherwise, the tag 'specification misalignment' could be used in the case XSD seams correctly aligned with the spec but something is wrong inside the spec.

**Comment by realholgi** [comment]
Ok, then its a  BUG.

**Comment by victorDARDENNE** [comment]
Fixed - [e662e76](https://github.com/ITxPT/S02/commit/e662e76f08d16af48025b808e14acfc765f6fccb)
